gui/tools/Helper.h

```diff
@@ -0,0 +1,227 @@
+#include <algorithm>
+#include <vector>
+#include <sstream>
+#include <string>
+
+// VTK includes
+#include <vtksys/SystemTools.hxx>
+#include "vtkMatrix3x3.h"
+#include "vtkMatrix4x4.h"
+#include "vtkNew.h"
+#include "vtkTransform.h"
+
+namespace help
+{
+  //----------------------------------------------------------------------------
+  // Description
+  // Split string by separated char
+  static void SplitString(const std::string &s, char delim,
+                          std::vector<std::string> &elems)
+  {
+    std::stringstream ss(s);
+    std::string item;
+    while (std::getline(ss, item, delim))
+      {
+      elems.push_back(item);
+      }
+  }
+
+  //----------------------------------------------------------------------------
+  // Description
+  // Get the filename path (copy from vtk in order to allow debug)
+  static std::string GetFilenamePath(const std::string& filename)
```

Comment:
If we need this still we should be using something from kwsys (in vital) or something from VTK or Qt.